compiler/rustc_hir_analysis/src/astconv/mod.rs

```diff
@@ -2395,13 +2395,22 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
                                     tcx,
                                     infcx.fresh_substs_for_item(DUMMY_SP, impl_def_id),
                                 );
+                                let universe = infcx.create_next_universe();
```

Comment:
I guess we don't need to make a universe unless we need it, but also we're on the error path, so it doesn't matter here.